test/common.js

```diff
@@ -211,6 +219,32 @@ export function getContainer({ parent, shadow = false, slots = false } : {| pare
         };
     }
 
+    if (nested) {
+        const innerWrapper = document.createElement('inner-element-wrapper');
+        innerWrapper.setAttribute('id', 'inner-element-wrapper-id');
+
+        const customElementShadowRoot = customElement.shadowRoot;
```

Comment:
There is still some logic here that isn't working as expected. @gabrielo91 will continue to debug.